Lib/pydoc_data/topics.py

```diff
@@ -5602,7 +5604,7 @@ class of the instance or a *non-virtual base class* thereof. The
    3232235521
    >>>
    >>> width = 5
-   >>> for num in range(5,12):
+   >>> for num in range(5,12): 
```

Comment:
lint nit:
```suggestion
   >>> for num in range(5,12):
```

Reply:
I don't think this is caused by this PR.

Reply:
Yep, turns out it was caused by the refactoring in https://github.com/python/cpython/pull/129116. Adam will check it.

(This did also make its way into `main` during yesterday's release, but since tidied up in https://github.com/python/cpython/pull/130014.)

Reply:
Regenerating the file on the main branch puts the space back in.
Do you want me to remove it manually?
I could also not change this file in the PR and leave it to the release process, but I think it's good to have the change in the history.